ds-card-reader/res/devices/device-list.yaml

```
@@ -0,0 +1,14 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+DeviceList:
+  - Name: card-reader
+    ProfileName: rfid-card-reader
+    Protocols:
+      other:
+        Address: simple01
+        Port: 300
+    AutoEvents:
+      - Interval: 3s
+        OnChange: true
+        SourceName: status
```

This still has not been addressed. The properties must be camelCase including (except for the items under `protocols`).

```suggestion
deviceList:
  - name: card-reader
    profileName: rfid-card-reader
    protocols:
      other:
        Address: simple01
        Port: 300
    autoEvents:
      - interval: 3s
        onChange: true
        sourceName: status
```